### PR TITLE
#123 fix the mix test and implementation

### PR DESCRIPTION
**Pull request by jt3k**
also run uglify grunt-task

### CODE REVIEW

**Comment by jt3k** [comment]
[](https://github.com/bgrins/TinyColor/issues/123)

**Comment by jt3k** [comment]
bump

File: test/test.js [hunk 1]
```diff
@@ -721,7 +721,7 @@ test("Mix", function () {
```
Comment (bgrins) [inline test/test.js]: Can you also add an 'equal' case for two colors that was failing before your change and now is passing?  After the check on line 715 would be a good place for that

**Comment by bgrins** [comment]
As far as the tinycolor-min.js changes - I've been waiting for version updates to re-run the uglify task

**Comment by jt3k** [comment]
I do not quite understand. Do I have to do something here yet or not?

**Comment by bgrins** [comment]
Can you please revert the changes to tinycolor-min.js?  Once that's done it'll be ready to land

**Comment by jt3k** [comment]
now is ok ?

**Comment by bgrins** [comment]
Yes, thanks!

**Pull request merged**